src/2_4.md

```diff
@@ -87,14 +87,16 @@ fn description_machine<T: Robot>(machine: T) {
 
 C'est pas trop génial ? :waw:
 
-Revenons-en maintenant à notre problème initial : "comment faire avec 40 espèces d'animaux différentes" ? Je pense que vous commencez à voir où je veux en venir je présume ? Non ? Très bien, dans ce cas prenons un exemple :
+Attention! Avec l'arrivée du compilateur rustc 1.1, il y a eu quelques petits changements dans les cas de méthodes par défaut...
```

Comment:
I think it's useless to give comparisons like this. The best would be to just update the tutorial's explanations and/or the impacted code. I let you update. :-)